webhooks/virtualmachine/v1alpha1/validation/virtualmachine_validator_unit_test.go

```diff
@@ -410,11 +428,13 @@ func unitTestsValidateCreate() {
 		ctx = newUnitTestContextForValidatingWebhook(false)
 		oldFaultDomainsFunc = lib.IsWcpFaultDomainsFSSEnabled
 		oldImageRegistryFunc = lib.IsWCPVMImageRegistryEnabled
+		oldVMServiceBackupRestoreFunc = lib.IsVMServiceBackupRestoreFSSEnabled
```

Comment:
I think the recommended way is to use `os.Setenv` and `os.Unsetenv`, see [this](https://github.com/vmware-tanzu/vm-operator/blob/main/pkg/vmprovider/providers/vsphere/vmprovider_vm_utils_test.go#L70-L74) for an example.

Reply:
I noticed, and considered that.  One issue with that approach is the fact that we don't "reset" the values of method.  As such, we run the risk of causing other tests to fail.  IOW, this test will not leave the value of the env var was it was before the test execution.